__frzr-deploy

```diff
@@ -4,34 +4,101 @@ set -e
 set -o pipefail
 
 get_img_url() {
-    CHANNEL=$1
-
-    result=$(jq '[
-          sort_by(.created_at) |
-          reverse |
-          .[] |
-          { name: .name, prerelease: .prerelease, state: .assets[].state, url: .assets[].browser_download_url } |
-          select(.url|test("img")) |
-          select(.state=="uploaded")
```

Comment:
we need this check, otherwise frzr can select releases which are not yet finished uploading.